packages/webdriverio/src/utils/interception/devtools.js

```diff
@@ -76,21 +76,25 @@ export default class DevtoolsInterception extends Interception {
                         newBody = await overwrite(request, client)
                     }
 
+                    if (typeof newBody === undefined) {
+                        newBody = ''
+                    }
```

Comment:
I don't think this is necessary because `newBody` will never be undefined, see

```js
                if (overwrite !== undefined) {
                   let newBody = overwrite
```

Reply:
if `overwrite` is a function user can return `undefined` in line 76